### ISSUE TITLE
Single-page changelog

### ISSUE THREAD

**Issue by Chriscbr** [winglang/wing#2933]
### Feature Spec

A single-page list of changes (viewable on docs.winglang.io) covering the entire history of features and fixes added during Wing's development.

Examples:
* https://docs.modular.com/mojo/changelog.html
* https://github.com/nodejs/node/blob/main/CHANGELOG.md
* https://github.com/JetBrains/kotlin/blob/master/ChangeLog.md

### Use Cases

For users that want to quickly check at a glance a list of the latest features that have been added in the past (week, month, or other period). This information is already available through our git history and through our [GitHub releases page](https://github.com/winglang/wing/releases), but these sources are sometimes more cumbersome to view.

### Component

Documentation

### Community Notes

* Please vote by adding a 👍 reaction to the issue to help us prioritize.
* If you are interested to work on this issue, please leave a comment.

**Comment by ekeren** [comment]
marking this as P2, @Chriscbr let me know if you disagree

**Comment by Chriscbr** [comment]
@ekeren Yep I think for now this is a wish list item (not highest priority), but of course contributions are always welcomed 👍

**Comment by Chriscbr** [comment]
This could also be handy for Wing libraries.
